src/shortlog.js

```diff
@@ -40,8 +40,7 @@ module.exports = function * () {
     var repos = flagutil.computeReposFromFlag(argv.r);
     var emailFilter = argv.filter && new RegExp(argv.filter);
     var days = argv.days || 7;
-    /* eslint-enable no-undef */
-    var resultsByAuthor = Object.create(null);
+    var resultsByAuthor = {};
```

Comment:
Actually, [`Object.create(null)` is safer](https://davidwalsh.name/object-create-null), so I'd keep it. Not that I would ever bother to do that in my code, but if it's already there :man_shrugging: